mindsdb_native/libs/phases/type_deductor/type_deductor.py

```diff
@@ -334,7 +334,8 @@ def run(self, input_data):
                     if col_name not in self.transaction.lmd['predict_columns']:
                         if (self.transaction.lmd.get('tss', None) and
                                 self.transaction.lmd['tss']['is_timeseries'] and
-                                col_name in self.transaction.lmd['tss']['order_by']):
+                                (col_name in self.transaction.lmd['tss']['order_by'] or
+                                col_name in (self.transaction.lmd['tss']['group_by'] or []))):
```

Comment:
Why the extra `[]` at the end?

Reply:
`self.transaction.lmd['tss']['group_by'] or []` always evaluates to an iterable, whereas `self.transaction.lmd['tss']['group_by']` can evaluate to None

Reply:
Got it, thanks